### PR TITLE
Update agent docs to add conceal secrets flag

### PR DESCRIPTION
**Pull request by vaijab**

### CODE REVIEW

**Comment by vaijab** [comment]
@bradrydzewski friendly ping, since you merged the code already.

**Comment by gtaylor** [comment]
@vaijab What is the expected value of this env var? `true`? If so, can you add an "If `true`" prefix as per some of the other vars?

File: content/reference/configuration/agent.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ NAME                        | DESC
 `DRONE_PLUGIN_PULL`         | If `true`, plugin updates are pull automatically.
 `DRONE_PLUGIN_NAMESPACE`    | Plugin namespace in dockerhub, defaults to `plugins`.
 `DRONE_PLUGIN_PRIVILEGED`   | Space-separated list of plugins automatically granted privileged mode.
+`DRONE_CONCEAL_SECRETS`     | Conceal secrets from build logs.
```
Comment (gtaylor) [inline content/reference/configuration/agent.md]: What is the expected value of this env var? true? If so, can you add an "If true" prefix as per some of the other vars?

**Comment by vaijab** [comment]
@gtaylor updated.

**Pull request merged**